custom_components/aguaiot/const.py

```diff
@@ -130,6 +152,16 @@ class AguaIOTSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=NumberDeviceClass.TEMPERATURE,
     ),
+    NumberEntityDescription(
+        key="power_set",
+        name="Pellet Power",
+        native_step=1,
+    ),
+    NumberEntityDescription(
```

Comment:
How do you even control wood power? Also a fan forcing air through? And why does the stove have two separate power controls? I assume there can be only one active at the same time?

Reply:
In pellet mode, pellet power is controlled by pellet infeed speed and fresh air control.
In wood mode,  wood power controls front fan speed (3 modes, and the "pellet mode" fan speed is ignored) and maybe (i'm not sure) air intake. It also control air intake when gooing to RIS mode (target temp reached, goes to minimum power to keep combustion on and ready to restart when reaching low temp treshold).
As the stove switch automaticaly from pellet to wood and wood to pellet (by controlling flame temp and gas temp), there is always both power control available (no matter of mode), and in nobis app i can set both. The used one will depend on mode (fan or pellet). But both act on the same target temp.

In usage, I can open the stove door at any moment to place wood inside : when off and start fire manually like an "old school" fire pit (never done it...), place wood and start with pellet (lazy but easy :) ), or simply start in pellet mode, open door and trash some logs into the stove. In any case the stove detect when logs are burning enough and switch to wood mode.

To not, real_power_get and real_power_wood_get stay at their previous stated when switching mode. But it seams that real_power_enabled and real_power_wood_enabled registers change (that's why i have to set them force_enabled=True, otherwise they are not created).

When the stove goes out of wood (detect lower flame and fume temp), I get an alarm "wood reserve". If I do nothing and heating is still required, the stove will switch back to pellet.